hack/verify-generated-swagger-docs.sh

```diff
@@ -1,46 +0,0 @@
-#!/usr/bin/env bash
-
-# Copyright 2015 The Kubernetes Authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-# This script checks whether updating of swagger type documentation is needed or
-# not. We should run `hack/update-generated-swagger-docs.sh` if swagger type
-# documentation is out of date.
-# Usage: `hack/verify-generated-swagger-docs.sh`.
-
-set -o errexit
-set -o nounset
-set -o pipefail
-
-KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
-source "${KUBE_ROOT}/hack/lib/init.sh"
-
-kube::util::ensure_clean_working_dir
-
-_tmpdir="$(kube::realpath "$(mktemp -d -t "$(basename "$0").XXXXXX")")"
-git worktree add -f -q "${_tmpdir}" HEAD
-kube::util::trap_add "git worktree remove -f ${_tmpdir}" EXIT
-cd "${_tmpdir}"
-
-# Update the generated swagger docs
-hack/update-generated-swagger-docs.sh
-
-# Test for diffs
-diffs=$(git status --porcelain | wc -l)
-if [[ ${diffs} -gt 0 ]]; then
-  echo "Generated swagger type documentation is out of date:" >&2
```

Comment:
how is this verified now?

Reply:
worktree, run, diff

Reply:
who runs that and where?

Reply:
verify-codegen.sh is called from `make verify`

Reply:
and it runs 
```
export UPDATE_API_KNOWN_VIOLATIONS=true
"${KUBE_ROOT}/hack/update-codegen.sh" --verify-only "$@"
```

Reply:
I was not using latest bits on my local branch, that is why I didn't find the verify code

Reply:
yeah, man.  rebase your local tree :)